src/Annotation/README.md

```diff
@@ -0,0 +1,236 @@
+# Annotation feature
+
+This feature provides a system to declare annotations in docComment and 
+retrieve it into php classes.
+
+## Declaration 
+_Rules to be observed when declaring an annotation in the docComment_
+##### Identification
+The identification character of an annotation **MUST** be an `@`  
+e.g. `@Route`
+
+##### Annotation name
+The annotation name:
+- **MUST** start width uppercase. \[A-Z\].
+- Can only use alpha-numeric characters. [a-zA-Z]. It can't use special
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6283010)